ecs-cli/modules/cli/local/ps_app.go

```diff
@@ -75,33 +75,42 @@ func listContainers(c *cli.Context) ([]types.Container, error) {
 		if err != nil {
 			return nil, err
 		}
-		return listContainersWithFilters(filters.NewArgs(
+		containers, err := listContainersWithFilters(filters.NewArgs(
 			filters.Arg("label", fmt.Sprintf("%s=%s", converter.TaskDefinitionLabelValue, file)),
 			filters.Arg("label", fmt.Sprintf("%s=%s", converter.TaskDefinitionLabelType, localproject.LocalTaskDefType)),
 		))
+		logrus.Infof("Searching for containers matching --%s=%s", flags.TaskDefinitionFile, c.String(flags.TaskDefinitionFile))
```

Comment:
nit: if you move the log statement before calling `listContainersWithFilters` you can keep the return statement